src/modules/Dropdown/Dropdown.js

```diff
@@ -593,6 +599,9 @@ export default class Dropdown extends Component {
     // prevent selecting duplicate items when the dropdown is closed
     if (_.isNil(value) || !open) return
 
+    // prevent selecting item if maxSelections is reached
+    if (!_.isNil(maxSelections) && this.state.value.length >= maxSelections) return
```

Comment:
If we will set default value of `maxSelections` to `0`, we will can do `maxSelections > 0` instead of `_.isNil`. BTW, SUI makes [similar thing](https://semantic-ui.com/modules/dropdown.html#multiple-select-settings)

Reply:
This seems to break the propType check that I did to see if ```multiple``` and ```selection``` is also on the element. :/ Any ideas how I can still implement this propType check?

Reply:
@layershifter If I remove the check if it also has ```multiple``` and ```selection``` props when using ```maxSelections```, I can get it to work. Would this be reasonable?

Reply:
Oh, I see. I had similar situation with `Embed`. Let's go with `_.isNil`.